packages/react/src/contexts/ClerkContextProvider.tsx

```diff
@@ -1,7 +1,7 @@
 import { deriveState } from '@clerk/shared/deriveState';
 import { ClientContext, OrganizationProvider, SessionContext, UserContext } from '@clerk/shared/react';
 import type { ClientResource, InitialState, Resources } from '@clerk/types';
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
Nit: Your editor probably added this, throughout the file we use `React.` and even `React.useEffect`